screening/openg2p.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "OpenG2P",
+  "ownership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://docs.openg2p.org/"
+  },
+  "dependencies": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://docs.openg2p.org/"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "nonPIIAccessMechanism": "Yes"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "OpenG2P is a tool used by adopters, the OpenG2P foundation by itself does not store any PII"
```

Comment:
This answer is not valid here. If they claim to be privacy compliant, then they need to provide supporting evidence.

Reply:
Supporting evidence showing OpenG2P is privacy compliant can now be found in this [commit](https://github.com/unicef/publicgoods-candidates/pull/112/commits/a7db56c683f65fdc3ed8db3a42cee52bd3e561a1).